app/scripts/metamask-controller.js

```diff
@@ -1006,7 +993,7 @@ module.exports = class MetamaskController extends EventEmitter {
    */
   setupUntrustedCommunication (connectionStream, originDomain) {
     // Check if new connection is blacklisted
-    if (this.blacklistController.checkForPhishing(originDomain)) {
+    if (this.phishingController.test(originDomain)) {
```

Comment:
Way to implement this wording change we'd agreed to as a team. This is fine.

Reply:
This was surprisingly tough to come up with since "blocklist" is overloaded considering our product's context. Happy to change this if we can think of something nicer-sounding that "phishing" `¯\_(ツ)_/¯`